common/defs/ethereum/networks.json

```diff
@@ -367,5 +367,25 @@
     "rskip60": false,
     "url": "https://pirl.io",
     "blockbook": []
+  },
+{
+    "chain": "volta",
+    "chain_id": 73799,
+    "slip44": 44,
+    "shortcut": "VT",
+    "name": "Volta",
+    "rskip60": false,
+    "url": "https://www.energyweb.org/",
+    "blockbook": []
+  },
+  {
+    "chain": "energyweb",
+    "chain_id": 246,
+    "slip44": 44,
```

Comment:
please use the assigned id based on SLIP44, which is `246` i believe?

Reply:
ok - all good